static/styles/widgets.css

```diff
@@ -114,6 +114,14 @@ button {
         &:hover {
             border-color: hsl(0, 0%, 60%);
         }
+
+        &:disabled {
+            position: relative;
+            z-index: -1;
+            filter: saturate(0);
+            background-color: hsl(0, 0%, 93%);
+            color: hsl(0, 3%, 52%);
+        }
```

Comment:
Do we need night theme CSS for this as well?

Reply:
Could you expand on this? I essentially copied the css for this disabled button
![image](https://user-images.githubusercontent.com/36671046/143787444-8f012589-aa8d-4864-9795-26306ff60754.png)